ansible/roles/ocp4-workload-debugging-workshop/tasks/workload.yml

```diff
@@ -22,11 +13,17 @@
     - merge
     definition: "{{ lookup('template', item ) | from_yaml }}"
   loop:
-  - ./templates/crd.j2
-  - ./templates/service_account.j2
-  - ./templates/cluster_role.j2
-  - ./templates/cluster_role_binding.j2
-  - ./templates/operator.j2
+    - ./templates/project.j2
```

Comment:
Same here. ./templates is not necessary. Also just a hint. If you use project.yaml.j2 then VSCode at least renders the YAML correctly...